enterprise/ha/src/test/java/org/neo4j/kernel/ha/TxPushStrategyConfigIT.java

```diff
@@ -282,24 +287,29 @@ public void failure( Throwable ex )
                 } );
 
         template.execute( transaction -> {
+            monitorListener.clear();
             db.createNode();
         } );
+
+        return monitorListener.missed();
     }
 
     private static class LastTxMapping
     {
         private final InstanceId serverId;
         private final long txId;
+        private final int missed;
 
-        public LastTxMapping( InstanceId serverId, long txId )
+        LastTxMapping( InstanceId serverId, long txId, int missed )
         {
             this.serverId = serverId;
             this.txId = txId;
+            this.missed = missed;
         }
 
         public void format( StringBuilder failures, long txId )
         {
-            if ( txId != this.txId )
+            if ( txId < this.txId - this.missed )
```

Comment:
Maybe (txId < thix.txId - this.missed || txId > this.txId) to keep the old semantics of failing on excessive amounts of transactions as well? Just an observation that the upper range was changed as well.